x-pack/plugins/enterprise_search/public/applications/app_search/components/settings/settings.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+import { EuiPageHeader, EuiPageHeaderSection, EuiPageContentBody, EuiTitle } from '@elastic/eui';
+
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { FlashMessages } from '../../../shared/flash_messages';
+
+export const Settings: React.FC = () => {
+  return (
+    <>
+      <SetPageChrome
+        trail={[
+          i18n.translate('xpack.enterpriseSearch.appSearch.settings.title', {
+            defaultMessage: 'Settings',
+          }),
+        ]}
+      />
```

Comment:
Not a blocker, just a reminder to myself - we should check with Nick and potentially also add viewed telemetry to this page at some point